lib/core/src/lib/form/services/widget-visibility.service.spec.ts

```diff
@@ -421,6 +421,30 @@ describe('WidgetVisibilityService', () => {
             expect(rightValue).toBe('dropdown_label');
         });
 
+        it('should getRightValue be able to return 0 when rightType is set to value', () => {
```

Comment:
rightType is set to value 
it should return 0 when rightValue is a number
it should return false when rightValue is a boolean
it should return empty string when rightValue is a string
it should return X when rightValue is undefined or null
it should return Y when rightValue is an object. ??